src/test/java/BaseTest.java

```diff
@@ -1,11 +1,89 @@
 import io.github.bonigarcia.wdm.WebDriverManager;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
 import org.openqa.selenium.chrome.ChromeOptions;
+import org.openqa.selenium.edge.EdgeDriver;
+import org.openqa.selenium.edge.EdgeOptions;
+import org.openqa.selenium.remote.DesiredCapabilities;
+import org.openqa.selenium.remote.RemoteWebDriver;
+import org.testng.annotations.AfterClass;
 import org.testng.annotations.BeforeSuite;
+import java.net.MalformedURLException;
+import java.net.URI;
+import java.net.URL;
+import java.util.HashMap;
 
-public class BaseTest {
 
+public class BaseTest {
+    public WebDriver driver ;
+    public String url = "https://qa.koel.app/";
     @BeforeSuite
-    static void setupClass() {
+    public void setupSuit() throws MalformedURLException {
+        String browser = System.getProperty("browser");
+        driver=setupBrowser(browser);
+    }
+    @AfterClass
+    public void closeBrowser() {
+        driver.quit();
+    }
+    WebDriver setupBrowser(String browser) throws MalformedURLException {
+        DesiredCapabilities caps = new DesiredCapabilities();
+        String gridURL = "http://192.168.1.105:4444";
+        switch (browser) {
+            case "chrome" -> {
```

Comment:
Hi,
Your homework 26 has been reviewed, but Test suite is failed! You got this problem, please correct it.

![image](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/144403954/9a47d4ba-42b2-48aa-802d-5b9d09ba404a)

Java Switch Statements Syntax:
```
switch(expression) {
  case x:
    // code block
    break;
  case y:
    // code block
    break;
  default:
    // code block
}
```
Please correct the switch-case structure correctly, and please send us a picture of you running Jenkin with your homework .
Best wishes, the team HW (Khanh,  khanh@testpro.io)

Reply:
![Screenshot 2023-09-11 234636](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/142187367/46e62900-7e5f-4364-a802-0b84ce6d6a26)
![Screenshot 2023-09-11 234659](https://github.com/Test-Pro-Bootcamp/A49-QA-Automation-Framework/assets/142187367/c6bcf176-f7db-4128-b077-73d697bbfbd7)

Reply:
Here are some screenshots of hw26 running on Jenkins